.eslintrc

```diff
@@ -29,6 +29,7 @@
   },
   "rules": {
     "backpack/use-tokens": 2,
+    "backpack/auto-import-tokens": 2,
```

Comment:
This config should live in `eslint-config-skyscanner` instead. Then we can upgrade this codebase to use the new version of that package in this PR.

Reply:
How should we deal with the native/web configuration though? We probably still need the config in the native folder.

Reply:
sure, but the defaults for:
```
"backpack/use-tokens": 2,
"backpack/auto-import-tokens": 2,
```

Already live in `eslint-config-skyscanner` so we should use em (thats what the shared config is for). Overrides for native can stay there as they are intentional

Reply:
Right, I vote for doing it on a follow up PR. Don't want to add more things to this one.